README.md

```diff
@@ -17,31 +17,40 @@ Publishers who just want to add the Hypothesis client to their pages and allow u
 
 First, you will need to create a couple of things on the Hypothesis service:
 
-1. A set of credentials (an "authclient") for managing annotations and users
-   associated with your domain.
+1. A set of credentials (a "client credentials" OAuth client) for managing users
+   and groups associated with your domain.
+
+2. A set of credentials (a "JWT bearer" OAuth client) for creating signed tokens that
+   the Hypothesis client can use to automatically log in as a particular user.
 
 2. A group associated with your domain to which users can post annotations.
    Annotations from this group will be displayed when the Hypothesis client
    loads on your pages.
 
 In [Hypothesis development environments](http://h.readthedocs.io/en/latest/developing/install/), you can do this using the `hypothesis` CLI tool:
 
-```sh
-# 1. Create an OAuth client which can manage annotations and users belonging
-# to partner.org
-./bin/hypothesis --dev authclient add --name Partner --authority partner.org
+1. Go to http://localhost:5000/admin/oauthclients and create a new OAuth client
+   with the type set to "client_credentials" and the authority set to "partner.org".
 
-# 2. Create an admin user for partner.org. This is needed because groups must
-# have a creator.
-./bin/hypothesis --dev user add --authority partner.org --username admin --email admin@localhost --password secret
+   This will give you a client ID and secret which can be used to create
+   accounts on the Hypothesis service via the API.
 
-# 3. Create the main group for annotations on partner.org
-./bin/hypothesis --dev groups add-publisher-group --authority partner.org --name Partner --creator admin
-```
+2. Create another OAuth client on the same page with the type set to
+   "jwt_bearer" and the authority also set to "partner.org"
+
+   This will give you a client ID and secret which can be used to generate
+   _grant tokens_ which can be used to identify the logged-in user to the
+   Hypothesis client.
+
+3. Create an admin user for partner.org:
```

Comment:
This is not strictly necessary anymore, we now allow groups without a creator. Groups without a creator cannot moderate their content, but other than that it should be all fine.
Just in case you want to have fewer steps to get started :)

(I guess the CLI for creating a group still needs the creator)

Reply:
Ah, cool. Thanks for the tip. I'll update this later.